Update c4-satellite-based-climate-data.md (#371)
MeteoSwiss/opendata: docs/c-climate-data/c4-satellite-based-climate-data.md

**Radiation**
- [Hourly, Daily, Monthly and Yearly satellite-based **Albedo**](./static/docs_img/static/docs_img/ProdDoc_RADIATION.pdf) (see [explanation of the term 'Albedo'](https://www.meteoswiss.admin.ch/weather/weather-and-climate-from-a-to-z/albedo.html))
- [Hourly, Daily, Monthly and Yearly satellite-based **Surface Incoming Solar Radiation with horizon**](./static/docs_img/static/docs_img/ProdDoc_RADIATION.pdf)
- [Hourly, Daily, Monthly and Yearly satellite-based **Surface Incoming Solar Radiation without horizon**](./static/docs_img/static/docs_img/ProdDoc_RADIATION.pdf)
- [Hourly, Daily, Monthly and Yearly satellite-based **Clear-sky Surface Incoming Solar Radiation with horizon**](./static/docs_img/static/docs_img/ProdDoc_RADIATION.pdf)
- [Hourly, Daily, Monthly and Yearly satellite-based **Clear-sky Surface Incoming Solar Radiation without horizon**](./static/docs_img/static/docs_img/ProdDoc_RADIATION.pdf)
  
**Diffuse Surface Incoming Solar Radiation**

> - *Hourly [to be verified]*, Daily, Monthly and Yearly satellite-based Diffuse Surface Incoming Solar Radiation *[Link auf ProdDoc_Radiation.pdf, M:\zue-prod\climate\basic_serv\spatial_analysis\Product_Docs\ProdDoc_Radiation.pdf]*

Reviewer: ```suggestion
- [Hourly, Daily, Monthly and Yearly satellite-based **Diffuse Surface Incoming Solar Radiation**](./static/docs_img/static/docs_img/ProdDoc_RADIATION.pdf)
```

sidebar_position: 4
---

# Spatial climate analyses – Radiation, Clouds, Land Surface Temperature

These grid data products are derived from MeteoSat satellite data. Ground observations are not included in the data. However, we use ground-based observations from the SwissMetNet and manual SYNOP cloud observation to quality check the satellite-based products. For product details and the product validation please refere to the documents below. 

**Radiation**
- [Hourly, Daily, Monthly and Yearly satellite-based **Albedo**](./static/docs_img/static/docs_img/ProdDoc_RADIATION.pdf) (see [explanation of the term 'Albedo'](https://www.meteoswiss.admin.ch/weather/weather-and-climate-from-a-to-z/albedo.html))

> - [Hourly, Daily, Monthly and Yearly **satellite-based Surface Incoming Solar Radiation with horizon**](./static/docs_img/static/docs_img/ProdDoc_RADIATION.pdf)

Reviewer: ```suggestion
- [Hourly, Daily, Monthly and Yearly satellite-based **Surface Incoming Solar Radiation with horizon**](./static/docs_img/static/docs_img/ProdDoc_RADIATION.pdf)
```

- [Hourly, Daily, Monthly and Yearly **satellite-based Surface Incoming Solar Radiation without horizon**](./static/docs_img/static/docs_img/ProdDoc_RADIATION.pdf)

**Clear-sky Surface Incoming Solar Radiation**

> - *Hourly [to be verified]*, Daily, Monthly and Yearly satellite-based Clear-sky Surface Incoming Solar Radiation with horizon *[Link auf ProdDoc_Radiation.pdf, M:\zue-prod\climate\basic_serv\spatial_analysis\Product_Docs\ProdDoc_Radiation.pdf]*

Reviewer: ```suggestion
- [Hourly, Daily, Monthly and Yearly satellite-based **Clear-sky Surface Incoming Solar Radiation with horizon**](./static/docs_img/static/docs_img/ProdDoc_RADIATION.pdf)
```

**Clear-sky Surface Incoming Solar Radiation without horizon**
- *Hourly [to be verified]*, Daily, Monthly and Yearly satellite-based Clear-sky Surface Incoming Solar Radiation without horizon *[Link auf ProdDoc_Radiation.pdf, M:\zue-prod\climate\basic_serv\spatial_analysis\Product_Docs\ProdDoc_Radiation.pdf]*
  
**Diffuse Surface Incoming Solar Radiation**
- *Hourly [to be verified]*, Daily, Monthly and Yearly satellite-based Diffuse Surface Incoming Solar Radiation *[Link auf ProdDoc_Radiation.pdf, M:\zue-prod\climate\basic_serv\spatial_analysis\Product_Docs\ProdDoc_Radiation.pdf]*

> **Diffuse Surface Incoming Solar Radiation**

Reviewer: ```suggestion
```

- *Hourly [to be verified]*, Daily, Monthly and Yearly satellite-based Diffuse Surface Incoming Solar Radiation *[Link auf ProdDoc_Radiation.pdf, M:\zue-prod\climate\basic_serv\spatial_analysis\Product_Docs\ProdDoc_Radiation.pdf]*

**Diffuse Surface Incoming Solar Radiation without horizon**

> - *Hourly [to be verified]* satellite-based Diffuse Surface Incoming Solar Radiation without horizon*[Link auf ProdDoc_Radiation.pdf, M:\zue-prod\climate\basic_serv\spatial_analysis\Product_Docs\ProdDoc_Radiation.pdf]*

Reviewer: ```suggestion
- [Hourly satellite-based **Diffuse Surface Incoming Solar Radiation without horizon**](./static/docs_img/static/docs_img/ProdDoc_RADIATION.pdf)
```

**Direct Surface Incoming Solar Radiation**

> - *Hourly [to be verified]*, Daily, Monthly and Yearly satellite-based Direct Surface Incoming Solar Radiation *[Link auf ProdDoc_Radiation.pdf, M:\zue-prod\climate\basic_serv\spatial_analysis\Product_Docs\ProdDoc_Radiation.pdf]*

Reviewer: ```suggestion
- [Hourly, Daily, Monthly and Yearly satellite-based **Direct Surface Incoming Solar Radiation**](./static/docs_img/static/docs_img/ProdDoc_RADIATION.pdf)
```

**Direct Surface Incoming Solar Radiation without horizon**
- *Hourly [to be verified]*, Daily, Monthly and Yearly satellite-based Direct Surface Incoming Solar Radiation without horizon*[Link auf ProdDoc_Radiation.pdf, M:\zue-prod\climate\basic_serv\spatial_analysis\Product_Docs\ProdDoc_Radiation.pdf]*

**Clear Sky Direct Surface Incoming Solar Radiation**
- *Hourly [to be verified]*, Daily, Monthly and Yearly satellite-based Clear-sky Direct Surface Incoming Solar Radiation *[Link auf ProdDoc_Radiation.pdf, M:\zue-prod\climate\basic_serv\spatial_analysis\Product_Docs\ProdDoc_Radiation.pdf]*

> **Clear Sky Direct Surface Incoming Solar Radiation without horizon**

Reviewer: ```suggestion
```

- *Hourly [to be verified]*, Daily, Monthly and Yearly satellite-based Clear-Sky Direct Surface Incoming Solar Radiation without horizon*[Link auf ProdDoc_Radiation.pdf, M:\zue-prod\climate\basic_serv\spatial_analysis\Product_Docs\ProdDoc_Radiation.pdf]*

> 

Reviewer: ```suggestion
```

**Direct Normal Surface Incoming Solar Radiation**
- *Hourly [to be verified]* satellite-based Direct Normal Surface Incoming Solar Radiation *[Link auf ProdDoc_Radiation.pdf, M:\zue-prod\climate\basic_serv\spatial_analysis\Product_Docs\ProdDoc_Radiation.pdf]*

**Direct Normal Surface Incoming Solar Radiation without horizon**

> - *Hourly [to be verified]* satellite-based Direct Normal Surface Incoming Solar Radiation without horizon*[Link auf ProdDoc_Radiation.pdf, M:\zue-prod\climate\basic_serv\spatial_analysis\Product_Docs\ProdDoc_Radiation.pdf]*

Reviewer: ```suggestion
- [Hourly satellite-based **Direct Normal Surface Incoming Solar Radiation without horizon**](./static/docs_img/static/docs_img/ProdDoc_RADIATION.pdf)
```

> 

Reviewer: ```suggestion
```

**Net Longwave Radiation**
- *Hourly [to be verified]*, Daily, Monthly and Yearly satellite-based Net Longwave Radiation *[Link auf ProdDoc_Radiation.pdf, M:\zue-prod\climate\basic_serv\spatial_analysis\Product_Docs\ProdDoc_Radiation.pdf]*

**Surface Radiation Budget**
- *Hourly [to be verified]*, Daily, Monthly and Yearly satellite-based Surface Radiation Budget *[Link auf ProdDoc_Radiation.pdf, M:\zue-prod\climate\basic_serv\spatial_analysis\Product_Docs\ProdDoc_Radiation.pdf]*

**Clouds**
- *Hourly [to be verified]* satellite-based Cloud Index *[Link auf ProdDoc_Radiation.pdf, M:\zue-prod\climate\basic_serv\spatial_analysis\Product_Docs\ProdDoc_Radiation.pdf]*
  
**Cloud Fractional Cover**
- *Hourly [to be verified]*, Daily, Monthly and Yearly satellite-based Cloud Fractional Cover *[Link auf ProdDoc_CLOUDS.pdf, M:\zue-prod\climate\basic_serv\spatial_analysis\Product_Docs\ProdDoc_CLOUDS.pdf]*

**Cloud Mask**
- *Hourly [to be verified]* satellite-based Cloud Mask *[Link auf ProdDoc_CLOUDS.pdf, M:\zue-prod\climate\basic_serv\spatial_analysis\Product_Docs\ProdDoc_CLOUDS.pdf]*

> 

Reviewer: ```suggestion
```

**Cloud Top Height**
- *Hourly [to be verified]* satellite-based Cloud Top Height *[Link auf ProdDoc_CLOUDS.pdf, M:\zue-prod\climate\basic_serv\spatial_analysis\Product_Docs\ProdDoc_CLOUDS.pdf]*

> 

Reviewer: ```suggestion
```

**Surface Outgoing Longwave Radiation**
- *Hourly [to be verified]*, Daily, Monthly and Yearly satellite-based Surface Outgoing Longwave Radiation *[Link auf ProdDoc_Radiation.pdf, M:\zue-prod\climate\basic_serv\spatial_analysis\Product_Docs\ProdDoc_Radiation.pdf]*

**Net Shortwave Radiation**
- *Hourly [to be verified]*, Daily, Monthly and Yearly satellite-based Net Shortwave Radiation *[Link auf ProdDoc_Radiation.pdf, M:\zue-prod\climate\basic_serv\spatial_analysis\Product_Docs\ProdDoc_Radiation.pdf]*

**Net Longwave Radiation**
- *Hourly [to be verified]*, Daily, Monthly and Yearly satellite-based Net Longwave Radiation *[Link auf ProdDoc_Radiation.pdf, M:\zue-prod\climate\basic_serv\spatial_analysis\Product_Docs\ProdDoc_Radiation.pdf]*

**Surface Radiation Budget**
- *Hourly [to be verified]*, Daily, Monthly and Yearly satellite-based Surface Radiation Budget *[Link auf ProdDoc_Radiation.pdf, M:\zue-prod\climate\basic_serv\spatial_analysis\Product_Docs\ProdDoc_Radiation.pdf]*

> **Cloud Index**

Reviewer: ```suggestion
**Clouds**
```

sidebar_position: 4
---

# Spatial climate analyses – Radiation, Clouds, Land Surface Temperature

These grid data products are derived from MeteoSat satellite data. Ground observations are not included in the data. However, we use ground-based observations from the SwissMetNet and manual SYNOP cloud observation to quality check the satellite-based products. For product details and the product validation please refere to the documents below. 

> **Albedo**

Reviewer: ```suggestion
**Albedo** (see [explanation of the term](https://www.meteoswiss.admin.ch/weather/weather-and-climate-from-a-to-z/albedo.html))
```

- [Hourly, Daily, Monthly and Yearly **satellite-based Albedo**](./static/docs_img/static/docs_img/ProdDoc_RADIATION.pdf)

> **Surface Incoming Solar Radiation**

Reviewer: ```suggestion
```

> - *Hourly [to be verified]*, Daily, Monthly and Yearly satellite-based Surface Incoming Solar Radiation with horizon *[Link auf ProdDoc_Radiation.pdf, M:\zue-prod\climate\basic_serv\spatial_analysis\Product_Docs\ProdDoc_Radiation.pdf]*

Reviewer: ```suggestion
- [Hourly, Daily, Monthly and Yearly satellite-based Surface Incoming Solar Radiation with horizon](./static/docs_img/static/docs_img/ProdDoc_RADIATION.pdf)
```
